Update index.html.md (#1579)
hashicorp/terraform-website: content/source/docs/cloud/agents/index.html.md

> You may choose to run multiple agents within your network for improved resiliency. If there are multiple agents available within an organization, Terraform Cloud will select the first available agent.

Reviewer: ```suggestion
You may choose to run multiple agents within your network, up to the organization's purchased agent limit. If there are multiple agents available within an organization, Terraform Cloud will select the first available agent within the target pool.
```

#### Resilience

> Presently, there isn't a provision for backup or standby agents. Agents should be run in a scheduler (Kubernetes, [Nomad](https://www.nomadproject.io/), etc) for resource management purposes.

Reviewer: ```suggestion
If an agent terminates abnormally, any running jobs can be restarted where they left off by resubmitting them. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
```

### Agent Capacity Usage

> Agents are considered active and count towards the organization's total agent allowance if they are in the **Idle**, **Busy**, or **Unknown** state. Agents which are in the **Errored** or **Exited** state do not count towards the organizations total agent capacity.

Reviewer: ```suggestion
Agents are considered active and count towards the organization's purchased agent capacity if they are in the **Idle**, **Busy**, or **Unknown** state. Agents which are in the **Errored** or **Exited** state do not count towards the organizations total agent capacity.
```

> Agents are considered active and count towards the organization's purchased agent capacity if they are in the **Idle**, **Busy**, or **Unknown** state. Agents which are in the **Errored** or **Exited** state do not count towards the organizations total agent capacity.

Reviewer 2: ```suggestion
Agents are considered active and count towards the organization's purchased agent capacity if they are in the **Idle**, **Busy**, or **Unknown** state. Agents which are in the **Errored** or **Exited** state do not count towards the organization's total agent capacity.
```
